src/Components/Artist/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -26,8 +27,17 @@ export class ArtistCollectionEntity extends React.Component<CollectionProps> {
   }
 
   render() {
-    const { price_guidance, slug, title } = this.props.collection
+    const {
+      artworks: { hits },
+      headerImage,
+      price_guidance,
+      slug,
+      title,
+    } = this.props.collection
     const formattedTitle = (title && title.split(": ")[1]) || title
+    const bgImages = map(hits, "image.url")
```

Comment:
what does passing the `"image.url"` string do exactly?

Reply:
The string is used to get a nested value of the object. The structure of the artwork is `{ ...artwork, image: { url: "string" } }`, by using[ lodash's `map`](https://lodash.com/docs/4.17.11#map) we can grab the values from an array of objects. This is the same as `pluck` in underscore.